config/jobs/periodic/knative/operator/main/operator-main.gen.yaml

```diff
@@ -0,0 +1,64 @@
+periodics:
+  - name: knative-operator-main-periodic 
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    cron: "20 0 * * *"
+    extra_refs:
+      - base_ref: main
+        org: ppc64le-cloud
+        repo: knative-upstream-ci
+        workdir: true
+    spec:
+      containers:
+        - image: quay.io/powercloud/knative-prow-tests:latest
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
+          command:
+            - runner.sh
+          args:
+            - bash 
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              TIMESTAMP=$(date +%s)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+
+              kubetest2 tf --powervs-image-name CentOS9-Stream\
+                --powervs-region syd --powervs-zone syd05 \
+                --powervs-service-id af3e8574-29ea-41a2-a9c5-e88cba5c5858 \
+                --powervs-ssh-key knative-ssh-key \
+                --ssh-private-key ~/.ssh/ssh-key \
+                --build-version $K8S_BUILD_VERSION \
+                --cluster-name knative-$TIMESTAMP \
+                --workers-count 2 \
+                --up --auto-approve --retry-on-tf-failure 5 \
+                --break-kubetest-on-upfail true \
+                --powervs-memory 32
+          
+              export KUBECONFIG="$(pwd)/knative-$TIMESTAMP/kubeconfig"
+          
+              grep -E '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' $(pwd)/knative-$TIMESTAMP/hosts > HOSTS_IP
+              source setup-environment.sh HOSTS_IP
+
+              git clone https://github.com/knative/operator.git
+              pushd operator
+              . /tmp/adjust.sh 
+              ./test/e2e-tests.sh --run-tests
+              popd
+
+              kubetest2 tf --powervs-region syd --powervs-zone syd05 \
+                --powervs-service-id af3e8574-29ea-41a2-a9c5-e88cba5c5858 \
```

Comment:
@Rajalakshmi-Girish TODO: make sure you add a pvsadm cleanup job for this workspace